src/controller.py

```diff
@@ -68,14 +82,30 @@ def start_game(self, player_names):
         self.game.start_game()
         self._switch_player()
 
-    def _switch_player(self):
-        self.current_player = self.players[next(self.name_cycle)]
-        logging.info(
-            f'Active player: {self.current_player.name}')
+    def send(self, command):
+        if command == 'quit':
```

Comment:
It is OK as for now. However, this line introduces a second layer of command parsing. We might need another method for 'controller layer commands' parsing. These commands could do nothing to the Game state, while doing something useful to the game session.